etc/c/odbc/sql.d

```diff
@@ -0,0 +1,1414 @@
+    /+ IN    +/ SQLSMALLINT Type,
+    /+ IN    +/ SQLSMALLINT SubType,
+    /+ IN    +/ SQLINTEGER  Length,
+    /+ IN    +/ SQLSMALLINT Precision,
+    /+ IN    +/ SQLSMALLINT Scale,
+    /+ INOUT +/ SQLPOINTER  Data,
+    /+ INOUT +/ SQLINTEGER  *StringLength,
+    /+ INOUT +/ SQLINTEGER  *Indicator
+);
+
+/+
+ ' ODBC v3.0+ ISO 92
+ ' Sets attributes that govern aspects of environments.
+ '
+ +/
+SQLRETURN SQLSetEnvAttr
+(
+    /+ IN  +/ SQLHENV    EnvironmentHandle,
+    /+ IN  +/ SQLINTEGER Attribute,
+    /+ IN  +/ SQLPOINTER Value,
+    /+ IN  +/ SQLINTEGER StringLength
+);
+
+/+
+ ' ODBC v3.0+ ISO 92
+ ' Sets attributes related to a statement.
+ '
+ +/
+SQLRETURN SQLSetStmtAttr
+(
+    /+ IN  +/ SQLHSTMT   StatementHandle,
+    /+ IN  +/ SQLINTEGER Attribute,
+    /+ IN  +/ SQLPOINTER Value,
+    /+ IN  +/ SQLINTEGER StringLength
+);
+
+/+
+ ' ODBC v1.0+ X/Open
+ ' Retrieves the following information about columns within a specified table:
+ '
+ ' 1) The optimal set of columns that uniquely identifies a row in the table.
+ ' 2) Columns that are automatically updated when any value in the row is updated by a transaction.
+ '
+ '
+ +/
+SQLRETURN SQLSpecialColumns
+(
+    /+ IN  +/ SQLHSTMT     StatementHandle,
+    /+ IN  +/ SQLUSMALLINT IdentifierType,
+    /+ IN  +/ SQLCHAR      *CatalogName,
+    /+ IN  +/ SQLSMALLINT  NameLength1,
+    /+ IN  +/ SQLCHAR      *SchemaName,
+    /+ IN  +/ SQLSMALLINT  NameLength2,
+    /+ IN  +/ SQLCHAR      *TableName,
+    /+ IN  +/ SQLSMALLINT  NameLength3,
+    /+ IN  +/ SQLUSMALLINT Scope,
+    /+ IN  +/ SQLUSMALLINT Nullable
+);
+
+/+
+ ' ODBC v1.0+ ISO 92
+ ' Retrieves a list of statistics about a single table and the
+ ' indexes associated with the table. The driver returns the
+ ' information as a result set.
+ '
+ ' -- Unique --
+ ' Type of index: SQL_INDEX_UNIQUE or SQL_INDEX_ALL.
+ '
+ +/
+SQLRETURN SQLStatistics
+(
+    /+ IN  +/ SQLHSTMT     StatementHandle,
+    /+ IN  +/ SQLCHAR      *CatalogName,
+    /+ IN  +/ SQLSMALLINT  NameLength1,
+    /+ IN  +/ SQLCHAR      *SchemaName,
+    /+ IN  +/ SQLSMALLINT  NameLength2,
+    /+ IN  +/ SQLCHAR      *TableName,
+    /+ IN  +/ SQLSMALLINT  NameLength3,
+    /+ IN  +/ SQLUSMALLINT Unique,
+    /+ IN  +/ SQLUSMALLINT Reserved
+);
+
+/+
+ ' OBDC v1.0+ X/Open
+ ' Returns the list of table, catalog, or schema names, and table
+ ' types, stored in a specific data source. The driver returns the
+ ' information as a result set.
+ '
+ +/
+SQLRETURN SQLTables
+(
+    /+ IN  +/ SQLHSTMT    StatementHandle,
+    /+ IN  +/ SQLCHAR     *CatalogName,
+    /+ IN  +/ SQLSMALLINT NameLength1,
+    /+ IN  +/ SQLCHAR     *SchemaName,
+    /+ IN  +/ SQLSMALLINT NameLength2,
+    /+ IN  +/ SQLCHAR     *TableName,
+    /+ IN  +/ SQLSMALLINT NameLength3,
+    /+ IN  +/ SQLCHAR     *TableType,
+    /+ IN  +/ SQLSMALLINT NameLength4
+);
+
+/+---------------------------+
+ | * Deprecated Functions *  |
+ +---------------------------+/

```

Comment:
deprecated {

would be a same to miss that and have another std.stream in phobos ;-)